tests/test_models.py

```diff
@@ -8,3 +14,84 @@ def test_resnet50_rcnn():
     model = keras_rcnn.models.ResNet50RCNN(inputs, 21, 300)
 
     model.compile(loss=["mse", "mse", "mse"], optimizer="adam")
+
+
+def test_rpn():
+    options = {
+        "activation": "relu",
+        "kernel_size": (3, 3),
+        "padding": "same"
+    }
+
+    image = keras.layers.Input((224, 224, 3))
+
+    y_true = keras.layers.Input((None, 4), name="y_true")
```

Comment:
This is a problem. You’d need to add a bounding box input to the `predict` method because the model will expect two inputs (an image and corresponding bounding boxes). A possible implementation is creating a custom layer that acts like `keras.layers.Input` during training but produces reference bounding boxes during test.